assets/package.json

```diff
@@ -16,7 +16,6 @@
     "phoenix_html": "file:../deps/phoenix_html",
     "phoenix_live_view": "file:../deps/phoenix_live_view",
     "phoenix": "file:../deps/phoenix",
-    "popper.js": "^1.15.0",
```

Comment:
I could be wrong, but neither the codebase, nor any of its dependencies use `popper.js`

Reply:
Could it be used by this or is that another popper implementation?

<img width="983" alt="Screenshot 2024-12-03 at 11 04 55" src="https://github.com/user-attachments/assets/c567bb69-b87d-4d03-a5a4-a02a9b707f01">

Reply:
It's still working so you are right.

Reply:
Yeah, I think it's Bootstrap `tooltip`, similar to what those copy buttons show:

![CleanShot 2024-12-03 at 11 23 22@2x](https://github.com/user-attachments/assets/3c67c613-be3c-4c2c-a46a-8d79c531bec8)